cltk/inflection/utils.py

```diff
@@ -78,6 +80,25 @@ def __init__(self, name: str, gender: Gender):
         self.name = name
         self.gender = gender
 
+    def set_void_declension(self, number_type, case_type):
+        """
+        >>> decl = DeclinableOneGender("armr", Gender.masculine)
+        >>> decl.declension
+        []
+        >>> decl.set_void_declension(Number, Case)
+        >>> decl.declension
+        [['', '', '', ''], ['', '', '', '']]
```

Comment:
I'm not sure the use of set_void_declension is obvious, please add a comment explaining here.
I'm note sure to make of the lists of empty strings. Why would these be preferable to just returning None, since, as I understand it, it would have the same effect.
Are there cases where one of those list elements might not be an empty string? If so, what would be there?

Reply:
Excellent, maybe update the doc comment with the reference.

Reply:
I will use the reference when I code the OldNorseNoun class. It is for a future pull request.